test/test.js

```diff
@@ -49,4 +50,19 @@ describe('GhReleases', function () {
         })
     })
   })
+
```

Comment:
This test can't pass because `electron-gh-releases-test` doesn't have a sub folder. Create a PR there to add it https://github.com/jenslind/electron-gh-releases-test/pull/1